test/SignalR.Orleans.Tests/OrleansHubLifetimeManagerTests.cs

```diff
@@ -1,30 +1,29 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.Threading.Tasks;
-using System.Threading.Tasks.Channels;
 using Microsoft.AspNetCore.SignalR;
 using Microsoft.AspNetCore.SignalR.Internal.Protocol;
-using Microsoft.AspNetCore.SignalR.Tests;
 using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Options;
-using SignalR.Orleans;
-using SignalR.Orleans.Tests;
+using Orleans;
+using SignalR.Orleans.Tests.Models;
+using System;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Channels;
 using Xunit;
 
 namespace SignalR.Orleans.Tests
 {
     public class OrleansHubLifetimeManagerTests : IClassFixture<OrleansFixture>
     {
-        private OrleansFixture _fixture;
+        private readonly OrleansFixture _fixture;
 
         public OrleansHubLifetimeManagerTests(OrleansFixture fixture)
         {
             this._fixture = fixture;
         }
 
         [Fact]
-        public async Task InvokeAllAsyncWritesToAllConnectionsOutput()
+        public async Task InvokeAllAsync_WritesTo_AllConnections_Output()

```

Comment:
I've updated the tests convention name since before it was annoying/difficult to read.

@galvesribeiro: What do you think?

Reply:
Those tests were mostly copy from Redis backplane. We can rename it, no problem.